packages/framework/src/Component/Doctrine/Exception/DuplicatedAliasException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Doctrine\Exception;
+
+use LogicException;
+use Throwable;
+
+class DuplicatedAliasException extends LogicException
+{
+    /**
+     * @param string $alias
+     * @param int|null $code
+     * @param \Throwable|null $previous
+     */
+    public function __construct(string $alias, ?int $code = 0, ?Throwable $previous = null)
```

Comment:
is the `$code` argument necessary? Usually, it's not used in extended exceptions (see other exceptions in the same namespace)

Reply:
Yeah you are right.